recipes/configs/qwen2/0.5B_lora.yaml

```diff
@@ -23,8 +23,7 @@ model:
   _component_: torchtune.models.qwen2.lora_qwen2_0_5b
   lora_attn_modules: ['q_proj', 'v_proj']
   apply_lora_to_mlp: False
-  apply_lora_to_output: False
-  lora_rank: 8
+  lora_rank: 64
```

Comment:
Any reason we don't just want to allow them to LoRA-ify the output, with default to False?

Reply:
0.5B and 1.5B have tied embeddings